network/src/commonMain/kotlin/com/wire/kalium/network/utils/NetworkUtils.kt

```diff
@@ -199,12 +200,24 @@ internal suspend fun handleUnsuccessfulResponse(
     result: HttpResponse
 ): NetworkResponse.Error {
     val status = result.status
+    val bodyText = try {
+        result.bodyAsText()
+    } catch (_: NoTransformationFoundException) {
+        // When the backend returns something that is not a JSON for whatever reason.
+        "NoTransformationFoundException"
+    } catch (_: DoubleReceiveException) {
+        // When the backend returns a JSON that cannot be parsed.
+        "DoubleReceiveException"
+    }
```

Comment:
Also seems impossible, as we do not read it twice.

Reply:
yes it seems impossible, but added it just in case